### PR TITLE
feat: 🎸(schema) adding more schema resources

### PR DESCRIPTION
**Pull request by MichaelDalyDev**

### CODE REVIEW

File: src/model/primitives/xhtml.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "type": "string",
+    "description": "Limited xhtml content (Escaped XHTML)"
```
Comment (fox1t) [inline src/model/primitives/xhtml.json]: I can't understand this primitive usage. Do we have some examples of what is allowed and what isn't?

Comment (MichaelDalyDev) [reply src/model/primitives/xhtml.json]: @fox1t No I don't have examples, this is up for discussion if you want to have it now. Perhaps this isn't required for the new version of HR. I just lifted it from the legacy RAML files and thought I should include it here.

Comment (fox1t) [reply src/model/primitives/xhtml.json]: OK. I think we can remove it for the moment and add it in future if needed. @jackcmeyer what do you think?

Comment (MichaelDalyDev) [reply src/model/primitives/xhtml.json]: sounds good. I will remove it and refresh the PR unless @jackcmeyer has other thoughts?

Comment (jackcmeyer) [reply src/model/primitives/xhtml.json]: @fox1t @MichaelDalyDev would this be used to store rich text content? Such as content from a rich text editor where we would want to preserve the html that is generated?

Comment (MichaelDalyDev) [reply src/model/primitives/xhtml.json]: @jackcmeyer that sounds like a valid use-case, but I cannot confirm how this type was used in the legacy system. Perhaps we can remove it for now, and introduce something more appropriate when we need it?

What do you think?

Comment (jackcmeyer) [reply src/model/primitives/xhtml.json]: I agree. We can add it when we need it!

Comment (MichaelDalyDev) [reply src/model/primitives/xhtml.json]: @jackcmeyer ok cool. I will remove it and refresh the PR.

**Comment by MichaelDalyDev** [comment]
@fox1t Not sure if you received the PR update notification yesterday evening, so wanted to post to confirm the xhtml schema object was removed and is ready for review.

**Comment by fox1t** [comment]
Thanks for pinging me @MichaelDalyDev! Merged.

**Pull request merged**